Up2dateService/Up2dateService/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the reason to add this file.
I guess it is not needed so far.

Reply:
removed